packages/mobile/src/transactions/saga.ts

```diff
@@ -54,18 +55,31 @@ export function* sendAndMonitorTransaction<T>(
     yield call(wrapSendTransactionWithRetry, txId, sendTxMethod)
     yield put(transactionConfirmed(txId))
 
+    txTime = Date.now() - txTime
     if (currency === CURRENCY_ENUM.GOLD) {
+      Logger.debug(
+        TAG + '@sendAndMonitorTransaction',
+        `send_gold_transaction_confirmed Performance (ms) = ` + txTime.toString()
+      )
+      CeloAnalytics.track(CustomEventNames.send_gold_transaction_confirmed, { txTime })
       yield put(fetchGoldBalance())
     } else if (currency === CURRENCY_ENUM.DOLLAR) {
-      CeloAnalytics.track(CustomEventNames.send_dollar_transaction_confirmed)
+      Logger.debug(
+        TAG + '@sendAndMonitorTransaction',
+        `send_dollar_transaction_confirmed Performance (ms) = ` + txTime.toString()
+      )
+      CeloAnalytics.track(CustomEventNames.send_dollar_transaction_confirmed, { txTime })
```

Comment:
note, you need to add txTime to the property whitelist in packages/mobile/src/analytics/constants.ts

Reply:
Thanks